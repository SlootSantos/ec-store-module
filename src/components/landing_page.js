import React, { Component } from 'react';

import '../styles/landing_page.css';
import coffee from '../assets/coffee.svg';
import star from '../assets/star.png';

class LandingPage extends Component {

  render() {
    return (
      <div className="main_container">
        <div className="page_one full_page">
          <div className="title">
            <span>Coffee</span>
            <img src={coffee} alt="" height="50"/>
            <span>BlaBla</span>
          </div>
        </div>
        <div className="page_two full_page">
        </div>
      </div>
    );
  }
};

export default LandingPage;
