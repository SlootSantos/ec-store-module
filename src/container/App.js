import React, { Component } from 'react';

// import connector
// redux action binder, to make actions available in component
// Link from router
// import { connect } from 'react-redux';
// import { bindActionCreators } from 'redux';


import LandingPage from '../container/landing_page'
import '../styles/App.css';

// import { fetchAllProducts } from '../actions/fetch_all_products';
// import { getCart } from '../actions/get_cart';

class App extends Component {
  // constructor(props) {
  //   super(props)
  //
  //   // this.props.fetchAllProducts();
  //   // this.props.getCart();
  // }

  // renderProductsToDom(products) {
  //
  //   if (!products || !products.length) return;
  //
  //   return products.map(product => {
  //     let { id, image_url } = product;
  //     console.log(product);
  //     return (
  //       <ProductTile key={id} product={product} img={image_url}/>
  //     );
  //   });
  // }


  render() {
    return (
      <div className="App">
          <LandingPage></LandingPage>
      </div>
    );
  }
}

// map state to props
// function mapStateToProps({ products, cart }) {
//
//   return {
//     products,
//     cart
//   };
// }

// map dispatch to props
// function mapDispatchToProps(dispatch) {
//   return bindActionCreators({ fetchAllProducts, getCart }, dispatch);
// }

// export default connect(mapStateToProps, mapDispatchToProps)(App);


export default App;
