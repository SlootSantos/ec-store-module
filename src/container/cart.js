import React, { Component } from 'react';
import { Link } from 'react-router-dom';
// import connector
// redux action binder, to make actions available in component
import { connect } from 'react-redux';
import { bindActionCreators } from 'redux';

import { getCart } from '../actions/get_cart';

import BackButton from '../components/back_button';

import '../styles/cart/cart.css';


class Cart extends Component {
  constructor(props) {
    super(props)

    this.props.getCart();
  }

  renderTable(elements) {
    return elements.items
      .map(item =>
        <tr key={item.id} className="table__details">
          <td>{ item.name }</td>
          <td>{ item.unit_price }</td>
          <td>{ item.quantity }</td>
          <td>{ item.value.amount }</td>
        </tr>
      );
    }

  renderTotals(amount) {
   return (
    <div className="cart__cart-totals">
      <table>
        <tbody>
          <tr>
            <th>Waren</th>
            <td>{ amount.amount }</td>
          </tr>

          <tr>
            <th>Shipping</th>
            <td>{ amount.amount }</td>
          </tr>

          <tr>
            <th>Gesamt</th>
            <td>{ amount.amount }</td>
          </tr>
        </tbody>
      </table>
    </div>
    );
  }

  render() {
    window.scrollTo(0, 0);


    return (
      <div className="cart">
        <BackButton />
        <h1 className="header_title">Dein Einkaufswagen</h1>

        { (!this.props.cart || !this.props.cart.items) &&
          <div>
            <div className="cart__empty">
              Du hast noch keinen Kaffee im Einkaufswagen...
            </div>
          </div>
        }

        { (this.props.cart && this.props.cart.items) &&
          <div className="cart__table-wraper">
            <table className="cart__cart-table">
              <tbody>
                <tr>
                  <th>Produkt</th>
                  <th>Preis</th>
                  <th>Anzahl</th>
                  <th>Gesamt</th>
                </tr>

                { this.renderTable(this.props.cart) }

              </tbody>
            </table>


            <Link className="btn btn-primary reverse" to="/shop/checkout">
             Kaufen
            </Link>
          </div> }
      </div>
    );
  }
}


// map state to props
function mapStateToProps({ cart }) {
  return {
    cart
  };
}

// map dispatch to props
function mapDispatchToProps(dispatch) {
  return bindActionCreators({ getCart }, dispatch);
}

export default connect(mapStateToProps, mapDispatchToProps)(Cart);
