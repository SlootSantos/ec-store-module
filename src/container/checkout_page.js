import React, { Component } from 'react';
import PropTypes from 'prop-types';
// import connector
// redux action binder, to make actions available in component
import { connect } from 'react-redux';
import { bindActionCreators } from 'redux';
import { Link } from 'react-router-dom';

import { getCart } from '../actions/get_cart';
import { checkoutCart } from '../actions/checkout_cart';

import OrderOverview from '../components/order_overview';
import SepaForm from '../components/payment_forms/sepa_form';
import CreditcartForm from '../components/payment_forms/credit-card_from';
import PaymentOverlay from '../components/payment_overlay';
import BackButton from '../components/back_button';

import PaypalLogo from '../assets/paypal.png';
import SepaLogo from '../assets/sepa.png';
import CreditLogo from '../assets/creditcards.png';
import Widerrufsformular from '../assets/Widerrufsformular.pdf';


import '../styles/checkoutpage/checkoutpage.css';

class CheckoutPage extends Component {
  constructor() {
    super();

    this.state = {
      showDifAdd: false,
      processPayment: false
    };
  }
  componentWillMount() {
    this.customerData = {};
    this.shippingData = {};
    this.paymentData = {};
  }

  setCusData(event, fieldName) {
    this.customerData[fieldName] = event.target.value;
  }

  setShipData(event, fieldName) {
    this.shippingData[fieldName] = event.target.value;
  }

  setDifAddress(event) {
    this.setState({
      showDifAdd: event.target.checked
    });
  }

  paymentDataHandler(event, fieldName) {
    this.paymentData[fieldName] = event.target.value;
  }

  createCheckoutObject() {
    this.setState({
      processPayment: true
    });
    const { customerData, paymentData, shippingData } = this;
    const cd = customerData;

    const data = {
      customer: {
        first_name: cd.firstname,
        last_name: cd.lastname,
        email: cd.email,
      },
      billing: {
        first_name: cd.firstname,
        last_name: cd.lastname,
        line_1: cd.line_1,
        line_2: cd.line_2,
        county: cd.county,
        postcode: cd.postcode,
        country: 'DE'
      },
      payment: paymentData
    };

    data.payment.amount_int = Math.round((this.props.cart.value.val_int) * 100) / 100;
    // data.payment.amount_int = Math.round((this.props.cart.value.val_int + 499) * 100) / 100;
    data.payment.currency = 'EUR';
    data.payment.type = this.state.paymentType;

    if (data.payment.type === 'credit') {
      const { expiryDate } = data.payment;

      if (!expiryDate.match(/[0-9]{2}[ -|/|.][0-9]{4}/g)) { return }

      const splitDate = expiryDate.match(/\./g)
      ? expiryDate.split('.')
      : expiryDate.split('/');

      delete data.payment.expiryDate;

      data.payment = Object.assign(data.payment, {
        exp_month: splitDate[0],
        exp_year: splitDate[1]
      });
    }

    data.shipping = this.state.showDifAdd
    ? shippingData
    : data.billing;

    checkoutCart(data)
    .then((res) => {
      if (res.data.errors) {
        this.props.history.push('/shop/checkout/fail');
      } else {
        this.props.getCart();
        this.props.history.push('/shop/checkout/success');
      }
  })
    .catch(() => this.props.history.push('/shop/checkout/fail'));
  }

  render() {
    return (
      <div className="checkout">
        <BackButton />
        { this.state.processPayment && <PaymentOverlay />}
        <h1 className="header_title">Dein Checkout</h1>
        {/* <PaypalButton id="button" /> */}
        <div className="checkout content">
          <div className="form-wrapper">
            <h1>Rechnungsadresse</h1>
            <form action="">
              <div className="checkout__input double">
                <span className="input__title">Vorname *</span>
                <input type="text" placeholder="Vorname" onChange={e => this.setCusData(e, 'firstname')} />
              </div>
              <div className="checkout__input double">
                <span className="input__title">Nachname *</span>
                <input type="text" placeholder="Nachname" onChange={e => this.setCusData(e, 'lastname')} />
              </div>
              <div className="checkout__input single">
                <span className="input__title">E-Mail Adresse *</span>
                <input type="text" placeholder="E-Mail Adresse" onChange={e => this.setCusData(e, 'email')} />
              </div>
              <div className="checkout__input double">
                <span className="input__title">Straße, Hausnr. *</span>
                <input type="text" placeholder="Straße, Hausnr." onChange={e => this.setCusData(e, 'line_1')} />
              </div>
              <div className="checkout__input double">
                <span className="input__title">Adresszusatz</span>
                <input type="text" placeholder="Adresszusatz" onChange={e => this.setCusData(e, 'line_2')} />
              </div>
              <div className="checkout__input double">
                <span className="input__title">Postleitzahl *</span>
                <input type="text" placeholder="Postleitzahl" onChange={e => this.setCusData(e, 'postcode')} />
              </div>
              <div className="checkout__input double">
                <span className="input__title">Stadt *</span>
                <input type="text" placeholder="Land" onChange={e => this.setCusData(e, 'county')} />
              </div>
            </form>
          </div>


          <div className="form-wrapper">
            <h1>An eine andere Adresse liefern?</h1>

            <input type="checkbox" className="checkout__show-dif-address" onClick={e => this.setDifAddress(e)} />

            { this.state.showDifAdd &&
            <form action="">
              <div className="checkout__input double">
                <span className="input__title">Vorname *</span>
                <input type="text" placeholder="Vorname" onChange={e => this.setShipData(e, 'firstname')} />
              </div>
              <div className="checkout__input double">
                <span className="input__title">Nachname *</span>
                <input type="text" placeholder="Nachname" onChange={e => this.setShipData(e, 'lastname')} />
              </div>
              <div className="checkout__input single">
                <span className="input__title">E-Mail Adresse *</span>
                <input type="text" placeholder="E-Mail Adresse" onChange={e => this.setShipData(e, 'email')} />
              </div>
              <div className="checkout__input double">
                <span className="input__title">Straße, Hausnr. *</span>
                <input type="text" placeholder="Straße, Hausnr." onChange={e => this.setShipData(e, 'line_1')} />
              </div>
              <div className="checkout__input double">
                <span className="input__title">Adresszusatz</span>
                <input type="text" placeholder="Adresszusatz" onChange={e => this.setShipData(e, 'line_2')} />
              </div>
              <div className="checkout__input double">
                <span className="input__title">Postleitzahl *</span>
                <input type="text" placeholder="Postleitzahl" onChange={e => this.setShipData(e, 'postcode')} />
              </div>
              <div className="checkout__input double">
                <span className="input__title">Stadt *</span>
                <input type="text" placeholder="Land" onChange={e => this.setShipData(e, 'county')} />
              </div>
            </form>}
          </div>
        </div>

        <OrderOverview cart={this.props.cart} />

        <div className="checkout__payment">
          <h1>Bezahlung</h1>

          <form className="checkout__payment-provider">
            <div>

              <div className="checkout__payment-provider-wrapper">
                <input id="paymentPaypal" type="radio" className="checkout__show-dif-address" value="paypal" name="payment" onClick={e => this.setState({ paymentType: e.target.value })} />
                <label htmlFor="paymentPaypal">
                  <img src={PaypalLogo} alt="" />
                </label> <br />
              </div>

              <div className="checkout__payment-provider-wrapper">
                <input id="paymentSepa" type="radio" className="checkout__show-dif-address" value="sepa" name="payment" onClick={e => this.setState({ paymentType: e.target.value })} />
                <label htmlFor="paymentSepa">
                  <img src={SepaLogo} alt="" />
                </label> <br />
              </div>

              <div className="checkout__payment-provider-wrapper">
                <input id="paymentCredit" type="radio" className="checkout__show-dif-address" value="credit" name="payment" onClick={e => this.setState({ paymentType: e.target.value })} />
                <label htmlFor="paymentCredit">
                  <img src={CreditLogo} alt="" />
                </label>
              </div>
            </div>
          </form>

          { this.state.paymentType === 'sepa' && <SepaForm onChangeHandler={this.paymentDataHandler.bind(this)} /> }
          { this.state.paymentType === 'credit' && <CreditcartForm onChangeHandler={this.paymentDataHandler.bind(this)} /> }
        </div>

        <div className="checkout__agreement">
          <input type="checkbox" className="checkout__show-dif-address" onClick={() => this.setState({ agreed: !this.state.agreed })} />
          <span>&nbsp;Mit deiner Bestellung erklärst du dich mit unseren <Link to="/agb">Allgemeinen Geschäftsbedingungen</Link> und <Link to={Widerrufsformular} target="_blank">Widerrufsbestimmungen</Link> einverstanden.</span>
        </div>
        <button className={`btn btn-primary reverse ${!this.state.agreed ? 'disabled' : ''}`} disabled={!this.state.agreed} onClick={() => this.createCheckoutObject()}>Kostenpflichtig bestellen</button>
      </div>
    );
  }
}

CheckoutPage.propTypes = {
  getCart: PropTypes.func.isRequired,
  cart: PropTypes.shape({
    value: PropTypes.shape({
      val_int: PropTypes.number
    })
  }).isRequired,
  history: PropTypes.shape({
    push: PropTypes.func
  }).isRequired
};

// map state to props
function mapStateToProps({ cart }) {
  return {
    cart
  };
}

// map dispatch to props
function mapDispatchToProps(dispatch) {
  return bindActionCreators({ getCart }, dispatch);
}

export default connect(mapStateToProps, mapDispatchToProps)(CheckoutPage);
