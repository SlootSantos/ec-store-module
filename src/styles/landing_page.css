.main_container {
  background: white; }

.full_page {
  height: 100vh;
  width: 100vw;
  background: white;
  display: flex;
  justify-content: center;
  align-items: center; }
