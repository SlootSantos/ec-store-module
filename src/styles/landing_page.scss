.main_container {
  background: white;
}

.full_page {
  height: 100vh;
  width: 100vw;

  &:first-child {
    height: 50vh;
    border-bottom: 1px solid white;
  }

  &.page_one {
    // background: url('http://wallpapercave.com/wp/E69rQhT.jpg');
    // background-repeat: no-repeat;
    // background-size: cover;
    background: white;
    display: flex;
    justify-content: center;
    align-items: center;

    .title {
      span {
        color: #373737;
        letter-spacing: 3px;
        font-size: 30px;
      }

      img {
        margin: 0 45px 0 40px;
      }
    }

    img {

    }
  }

  &.page_two {
    background-image: url('../assets/star.png');
  }
}
