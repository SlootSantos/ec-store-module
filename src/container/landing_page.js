import React from 'react';

// import full pages
import FullpageOne from '../components/fullpage_one';
import FullPageTwo from '../components/fullpage_two';
import FullPageThree from '../components/fullpage_three';
import FullPageMantra from '../components/fullpage_mantra';

import Espresso from '../assets/Espresso_white.svg';
import Classic from '../assets/CaPhePhin_white.svg';
import Filter from '../assets/FrenchPress_white.svg';
import VietFilter from '../assets/viet.Filter.svg';
import Milk from '../assets/milk.svg';

import '../styles/landingpage/landing_page.css';

export const mockData = [
  {
    name: 'Cà Phê Phin',
    id: '256acf70-cd6f-4406-af71-5d52523ad329',
    img_url:
      'https://s3.eu-central-1.amazonaws.com/www.vocoffee.de/traditionell_vietnamesischer_kaffee_bohnen_ca_phe_phin%400%2C5x.png',
    imgSrc: Classic,
    description:
      'Der <b>traditionell vietnamesische Kaffee</b> zeichnet sich durch seine starke Röstung aus, die dem Kaffee einen sehr kräftigen Geschmack gibt. Traditionell wird er mit gesüßter Kondensmilch auf Eis serviert, die die starke Röstung ausbalanciert und die Schokonoten der Bohnen hervorhebt und den Kaffee cremig macht.',
    bullets: [
      'Kräftiges Aroma',
      'Schokoladige Noten',
      '70% Arabica, 30 % Robusta'
    ],
    price: '€8,90',
    quantity: '250g'
  },
  {
    name: 'Thanh Trúc',
    id: '85e048e5-b4f1-49a5-a466-d52b01646729',
    img_url:
      'https://s3.eu-central-1.amazonaws.com/www.vocoffee.de/filterkaffee_gemahlen_thanh_truc%400%2C5x.png',
    imgSrc: Filter,
    description:
      'Unser <b>Filterkaffee</b> ist ein sortenreiner Kaffee aus 100% Bourbon Bohnen, eine sehr edle Sorte von Arabica Bohnen. Bourbon Bohnen sind bekannt dafür, dass sie eine feine komplexe Säurestruktur haben, die den Kaffee bekömmlich und fein machen.',
    bullets: [
      'Ausgewogenge, geringe Säure',
      'Feiner und milder Kaffee',
      '100% Bourbon Arabica'
    ],
    price: '€12,90',
    quantity: '250g'
  },
  {
    name: 'Dình Huy',
    id: 'ed0ee098-d5ba-409d-b2f4-c9610feca4b4',
    img_url:
      'https://s3.eu-central-1.amazonaws.com/www.vocoffee.de/espresso_gemahlen_dinh_huy%400%2C5x.png',
    imgSrc: Espresso,
    description:
      'Der <b>Espressokaffee</b> hat einen hohen Arabica Anteil, was dem Kaffee fruchtige Aromen verleiht. Da der vietnamesische Kaffee meist einen höheren Koffeinanteil hat, ist der Kaffee sehr kräftig und hat trotzdem eine wunderbare Crema.',
    bullets: ['Fruchtige Säure', 'Crema', '90% Arabica, 10% Robusta'],
    price: '€10,90',
    quantity: '250g'
  }
  // {
  //   name: 'V|O Coffee Filter',
  //   id: '28f3bae2-a791-403b-b769-00ea6ddeac5f',
  //   img_url:
  //     'https://s3.eu-central-1.amazonaws.com/www.vocoffee.de/vietnamesischer_filter%400%2C5x.png',
  //   imgSrc: VietFilter,
  //   description:
  //     'Um den traditionellen Kaffee „Ca Phe Phin“ zubereiten zu können, benötigt man den dazugehörigen Kaffeefilter „Phin“, der gleichzeitig auch Namensgeber des Kaffees ist.',
  //   bullets: [],
  //   price: '€8,00',
  //   quantity: 'Stück'
  // },
  // {
  //   name: 'Süße Kondensmilch',
  //   id: '2a21361e-15f3-4eee-ac70-e091444fc1c6',
  //   img_url:
  //     'https://s3.eu-central-1.amazonaws.com/www.vocoffee.de/kondensmilch%400%2C5x.png',
  //   imgSrc: Milk,
  //   description:
  //     'In Vietnam wird der traditionelle Kaffee „Cà Phê Phin“ mit der <b>gesüßten Kondensmilch</b> getrunken, um die dunkle Röstung der Bohnen auszubalancieren und die Schokonoten hervorzuheben.',
  //   bullets: [],
  //   price: '€2,00',
  //   quantity: 'Dose'
  // }
];

export function LandingPage() {
  return (
    <div className="main_container">
      <FullpageOne />
      <FullPageMantra />
      <FullPageTwo products={mockData} />
      <FullPageThree />
    </div>
  );
}
