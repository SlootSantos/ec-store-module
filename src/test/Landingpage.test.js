import React from 'react';
import ReactDOM from 'react-dom';
import { StaticRouter } from 'react-router';
import renderer from 'react-test-renderer';


import { LandingPage } from '../container/landing_page';
import FullPageOne from '../components/fullpage_one';
import FullPageTwo from '../components/fullpage_two';
import FullPageThree from '../components/fullpage_three';

test('render landingpage', () => {
  it('renders first fullpage', () => {
    const div = document.createElement('div');
    ReactDOM.render(
      (
        <StaticRouter context={{}}>
          <FullPageOne />
        </StaticRouter>
        ), div
      );
  });

  it('renders second fullpage', () => {
    const div = document.createElement('div');
    ReactDOM.render(
      (
        <StaticRouter context={{}}>
          <FullPageTwo />
        </StaticRouter>
        ), div
      );
  });

  it('renders third fullpage', () => {
    const div = document.createElement('div');
    ReactDOM.render(
      (
        <StaticRouter context={{}}>
          <FullPageThree />
        </StaticRouter>
        ), div
      );
  });

  it('renders entire landingpage', () => {
    const div = document.createElement('div');
    ReactDOM.render(
      (
        <StaticRouter context={{}}>
          <LandingPage />
        </StaticRouter>
        ), div
      );
  });
});


test('SNAPSHOT renders LandingPage correctily', () => {
  const imprSnap = renderer
    .create((
      <StaticRouter context={{}}>
        <LandingPage />
      </StaticRouter>))
    .toJSON();
  expect(imprSnap).toMatchSnapshot();
});
