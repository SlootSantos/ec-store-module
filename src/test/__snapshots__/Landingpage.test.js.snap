// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SNAPSHOT renders LandingPage correctily 1`] = `
<div
  className="main_container"
>
  <div
    className="full_page fp__one"
  >
    <div
      className="fp__one-layer"
    >
      <img
        alt="V|O COFFEE"
        src="vo_coffee_logo_white.svg"
      />
    </div>
  </div>
  <div
    className="full_page fp__mantra"
  >
    <div
      className="mantra-wrapper"
    >
      <p>
        Wenn Du gerne ein Stück von Vietnam und der Tradition des Landes in Deutschland erleben möchtest, probiere den vietnamesischen Kaffee und erlebe „Vietnam In A Cup“. 
        <br />
        <br />
        Für Updates folge uns auf unseren Social Media Kanälen und sei live dabei, wie wir wachsen und die Kaffeekultur verändern!
      </p>
      <div
        className="social-icons"
      >
        <img
          alt=""
          onClick={[Function]}
          src="facebook-logo.svg"
        />
        <img
          alt=""
          onClick={[Function]}
          src="instagram-logo.svg"
        />
      </div>
      <div
        className="mantra-bullets"
      >
        <h2>
          Warum V|O ?
        </h2>
        <ul>
          <li>
            Neue Lifestyle experience aus Vietnam
          </li>
          <li>
            Authentischer und traditionell vietnamesischer Kaffee
          </li>
          <li>
            Mehr Koffein als herkömmlicher Kaffee
          </li>
          <li>
            Hohe Qualität
          </li>
          <li>
            Andere Kaffeealternative
          </li>
          <li>
            Einzigartiger Geschmack
          </li>
        </ul>
      </div>
    </div>
  </div>
  <div
    className="full_page fp__two"
  >
    <div
      className="item-wrapper"
    >
      <div
        className="fp__two-item"
      >
        <div
          className="item-image-wrap"
        >
          <img
            alt=""
            className="fp__two-item-image"
            src="CaPhePhin.svg"
          />
        </div>
        <div
          className="item-text-wrap"
        >
          <h1>
            Cà Phê Phin
          </h1>
          <b>
            €8,90
          </b>
          <div
            className="fp__two-item-description"
          >
            <p>
              <span
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "Der <b>traditionell vietnamesische Kaffee</b> zeichnet sich durch seine starke Röstung aus, die dem Kaffee einen sehr kräftigen Geschmack gibt. Traditionell wird er mit gesüßter Kondensmilch auf Eis serviert, die die starke Röstung ausbalanciert und die Schokonoten der Bohnen hervorhebt und den Kaffee cremig macht.",
                  }
                }
              />
              <span
                className="fp__two-bullet"
              >
                Kräftiges Aroma
              </span>
              <span
                className="fp__two-bullet"
              >
                Schokoladige Noten
              </span>
              <span
                className="fp__two-bullet"
              >
                70% Arabica, 30 % Robusta
              </span>
              <span
                className="btn-wrapper"
              >
                <a
                  className="btn btn-primary"
                  href="/shop/Cà_Phê_Phin-256acf70-cd6f-4406-af71-5d52523ad329"
                  onClick={[Function]}
                >
                  Jetzt Bestellen!
                </a>
              </span>
            </p>
          </div>
        </div>
      </div>
      <div
        className="fp__two-item"
      >
        <div
          className="item-image-wrap"
        >
          <img
            alt=""
            className="fp__two-item-image"
            src="FrenchPress.svg"
          />
        </div>
        <div
          className="item-text-wrap"
        >
          <h1>
            Thanh Trúc
          </h1>
          <b>
            €12,90
          </b>
          <div
            className="fp__two-item-description"
          >
            <p>
              <span
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "Unser <b>Filterkaffee</b> ist ein sortenreiner Kaffee aus 100% Bourbon Bohnen, eine sehr edle Sorte von Arabica Bohnen. Bourbon Bohnen sind bekannt dafür, dass sie eine feine komplexe Säurestruktur haben, die den Kaffee bekömmlich und fein machen.",
                  }
                }
              />
              <span
                className="fp__two-bullet"
              >
                Ausgewogenge, geringe Säure
              </span>
              <span
                className="fp__two-bullet"
              >
                Feiner und milder Kaffee
              </span>
              <span
                className="fp__two-bullet"
              >
                100% Bourbon Arabica
              </span>
              <span
                className="btn-wrapper"
              >
                <a
                  className="btn btn-primary"
                  href="/shop/Thanh_Trúc-85e048e5-b4f1-49a5-a466-d52b01646729"
                  onClick={[Function]}
                >
                  Jetzt Bestellen!
                </a>
              </span>
            </p>
          </div>
        </div>
      </div>
      <div
        className="fp__two-item"
      >
        <div
          className="item-image-wrap"
        >
          <img
            alt=""
            className="fp__two-item-image"
            src="Espresso.svg"
          />
        </div>
        <div
          className="item-text-wrap"
        >
          <h1>
            Dình Huy
          </h1>
          <b>
            €10,90
          </b>
          <div
            className="fp__two-item-description"
          >
            <p>
              <span
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "Der <b>Espressokaffee</b> hat einen hohen Arabica Anteil, was dem Kaffee fruchtige Aromen verleiht. Da der vietnamesische Kaffee meist einen höheren Koffeinanteil hat, ist der Kaffee sehr kräftig und hat trotzdem eine wunderbare Crema.",
                  }
                }
              />
              <span
                className="fp__two-bullet"
              >
                Fruchtige Säure
              </span>
              <span
                className="fp__two-bullet"
              >
                Crema
              </span>
              <span
                className="fp__two-bullet"
              >
                90% Arabica, 10% Robusta
              </span>
              <span
                className="btn-wrapper"
              >
                <a
                  className="btn btn-primary"
                  href="/shop/Dình_Huy-ed0ee098-d5ba-409d-b2f4-c9610feca4b4"
                  onClick={[Function]}
                >
                  Jetzt Bestellen!
                </a>
              </span>
            </p>
          </div>
        </div>
      </div>
    </div>
    <div
      className="disclaimer"
    >
      Jeweils 250g verfügbar 
      <b>
        gemahlen
      </b>
       oder in 
      <b>
        Bohnen
      </b>
    </div>
  </div>
  <div
    className="full_page fp__three"
  >
    <div
      className="fp__three-headline"
    >
      <h1>
        V | O Crew
      </h1>
      <h4>
        Wir ♡ Kaffee
      </h4>
    </div>
    <div
      className="fp__three-introduction"
    >
      <p>
        <b>
          V|O Coffee
        </b>
         wurde im Sommer 2016 von Petra gegründet. Als einer der ersten Online-Anbieter für vietnamesischen Kaffee in Deutschland wollen wir unsere Liebe für Kaffee mit euch teilen und die Kaffeekultur in Deutschland bereichern.
        <br />
        <br />
        Seit der Gründung sind wir als Team gewachsen und arbeiten im Kernteam zu viert an dem Start-Up. Erste Erfolge konnten wir auch schon verzeichnen. Denn bei dem Rhein-Neckar-Start-Up-Weekend konnten wir uns den 3. Platz sichern und haben schon bei diversen Marketing-Events unseren Kaffee erfolgreich angeboten. Dabei wollen wir es natürlich nicht belassen, sondern noch viel mehr erreichen!
      </p>
    </div>
    <div
      className="fp__three-item-wrapper"
    >
      <div
        className="fp__three-items"
      >
        <div
          className="fp__three-items"
        >
          <div
            className="fp__three-item-bubble outter"
          >
            <div
              className="fp__three-item-bubble inner"
            >
              <img
                alt="showing"
                src="petra_web.jpg"
              />
            </div>
          </div>
        </div>
        <div
          className="description"
        >
          <h1>
            Petra
          </h1>
          <h4>
            The Chaperone
          </h4>
          <p
            dangerouslySetInnerHTML={
              Object {
                "__html": "Petra ist die Gründerin von V|O Coffee. Bereits vor ihrer Gründung war sie selbstständig und hat neben ihrem Englisch und BWL Studium an der Uni Mannheim viel im Sales und Merchandising Bereich gearbeitet. Als zertifizierte Barista ist ihr die Qualität bei Kaffee besonders wichtig. <br />Mit <b>V|O Coffee</b> kann sie nun ihre Kaffeeliebe und ein Stück ihrer vietnamesischen Kultur mit euch teilen.",
              }
            }
          />
        </div>
      </div>
      <div
        className="fp__three-items"
      >
        <div
          className="fp__three-items"
        >
          <div
            className="fp__three-item-bubble outter"
          >
            <div
              className="fp__three-item-bubble inner"
            >
              <img
                alt="showing"
                src="leo_web.jpg"
              />
            </div>
          </div>
        </div>
        <div
          className="description"
        >
          <h1>
            Leo
          </h1>
          <h4>
            The Accountant
          </h4>
          <p
            dangerouslySetInnerHTML={
              Object {
                "__html": "Wenn es um kaufmännische Entscheidungen geht, dann ist Leo unser Mann. Vor V|O Coffee hat er Akkreditivgeschäfte der BASF am Standort Hong Kong betreut und studiert jetzt Unternehmensjura an der Uni Mannheim.<br />Bei <b>V|O Coffee</b> ist er dabei, weil er an das Produkt glaubt und Petras Leidenschaft zu V|O ihn überzeugt hat.",
              }
            }
          />
        </div>
      </div>
      <div
        className="fp__three-items"
      >
        <div
          className="fp__three-items"
        >
          <div
            className="fp__three-item-bubble outter"
          >
            <div
              className="fp__three-item-bubble inner"
            >
              <img
                alt="showing"
                src="philipp_web.jpg"
              />
            </div>
          </div>
        </div>
        <div
          className="description"
        >
          <h1>
            Philipp
          </h1>
          <h4>
            The Creator
          </h4>
          <p
            dangerouslySetInnerHTML={
              Object {
                "__html": "Philipp ist der Kreative bei V|O und arbeitet als digitaler Mediendesigner in einer Werbeagentur. Durch sein Ehrenamt hat er ein Jahr in Vietnam gelebt, wo er das erste Mal den vietnamesischen Kaffee kennenlernte. <br />Durch <b>V|O Coffee</b> möchte er daran mitwirken, diesen auch in Deutschland bekannter zu machen und die Kaffeelandschaft zu bereichern.",
              }
            }
          />
        </div>
      </div>
      <div
        className="fp__three-items"
      >
        <div
          className="fp__three-items"
        >
          <div
            className="fp__three-item-bubble outter"
          >
            <div
              className="fp__three-item-bubble inner"
            >
              <img
                alt="showing"
                src="flo_square.jpg"
              />
            </div>
          </div>
        </div>
        <div
          className="description"
        >
          <h1>
            Flo
          </h1>
          <h4>
            The Nerd
          </h4>
          <p
            dangerouslySetInnerHTML={
              Object {
                "__html": "Flo ist unser IT-Guru. Schon in jungen Jahren hat er sich für das Coding interessiert. Heute arbeitet er als Freelancer im Software Engineering und hat u.a. bei Volkswagen, E.ON und AboutYou gearbeitet. Wie jeder andere Nerd auch ist Flo ein Kaffeetrinker. <br />Bei <b>V|O Coffee</b> kann er seine zwei größten Lieben vereinen.",
              }
            }
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
